packages/tracking/package.json

```diff
@@ -30,5 +30,8 @@
   "peerDependencies": {
     "@babel/runtime": "^7.12.5",
     "react": ">= 16.14.0"
+  },
+  "dependencies": {
+    "react-load-script": "^0.0.6"
```

Comment:
one of those old libraries, that are not maintained anymore: https://www.npmjs.com/package/react-load-script

maybe time to either look for a replacement or write a little helper ourselves?